src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = { ...state };
```

Comment:
The 'newState' object is being mutated across actions, which will cause all elements in 'listOfNewStates' to reference the same object. Instead, you should create a new copy of the state inside the 'oneAction' function for each action.